### PR TITLE
Add Day 1 Queries

### PR DESCRIPTION
**Pull request by ehmaddd**
In this PR, the following queries are added :

-  Create a database named vet_clinic.

-  Create a table named animals with the following columns:
  * id: integer
  * name: string
  * date_of_birth: date
  * escape_attempts: integer
  * neutered: boolean
  * weight_kg: decimal
----------------------------------------------------------------------------------------------
Copied the above SQL Queries into the schema.sql file.
----------------------------------------------------------------------------------------------
Inserted the following data :

-  Animal : 
   His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
-  Animal :
   Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
-  Animal :
   His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
-  Animal :
   Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
----------------------------------------------------------------------------------------------
Copied the above SQL Queries into the data.sql file
----------------------------------------------------------------------------------------------
Written the queries for the following quests :

-  Find all animals whose name ends in "mon".
-  List the name of all animals born between 2016 and 2019.
-  List the name of all animals that are neutered and have less than 3 escape attempts.
-  List the date of birth of all animals named either "Agumon" or "Pikachu".
-  List name and escape attempts of animals that weigh more than 10.5kg
-  Find all animals that are neutered.
-  Find all animals not named Gabumon.
-  Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)

The screenshot for the query results is here :
Screenshot

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1 +1,8 @@
-
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```
Comment (Ismailco) [inline queries.sql]: - [x] Greate work on selecting the names from the animals table however the date should be between '2016-01-01' AND '2019-31-12', kindly make sure to change the last date accordingly

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @ehmaddd, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ Clean code
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
- [x] Kindly consider removing the `Live Dome` section from your README file because this is a SQL project it will not be deployed

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (ehmaddd) [reply queries.sql]: Changes have been made as per instructions

File: README.md [hunk 2]
```diff
@@ -8,7 +8,6 @@
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
```
Comment (Gambit142) [inline README.md]: - [OPTIONAL] Great work removing the **_Live Demo Section_**. The technologies listed under the Tech Stac section were not used to build this project, especially React.js and Epress.js. Kindly remove them from the README. 👍

**Review by Gambit142 — APPROVED** [review]
Hi @ehmaddd 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- Everything that creates or changes the structure of the table was saved in the **_schema.sql_** file ✔️ 
- Everything that populates tables with data was saved in the **_data.sql_** file ✔️ 
- Everything that is a query that helps to analyze data should be saved in the **_queries.sql_** file ✔️ 

### Optional Suggestion
- Kindy consider the comment tagged _**[OPTIONAL]**_ under this review summary 👍 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (ehmaddd) [reply README.md]: Removed as per recommendations.
Regards

**Pull request merged**